src/App.css

```diff
@@ -50,8 +51,26 @@ tr {
 .footer {
   padding-top: 20px;
   text-align: center;
+  background-color: rgb(73, 65, 161);
+  color: white;
 }
 
+.card-container {
+  display: grid;
+  grid-template-columns: repeat(3, auto);
+}
 .card {
```

Comment:
This is the same name as Bootstrap's card componant even though it sets completely diffrent and non conflicting settings, which could be confusing. If you want a card in a diffrent size elsewhere you won't be able to benefit from Bootstrap's card styling without overriding this setting. I might make this look for a clasname like city-card or something